src/Error/Error.php

```diff
@@ -122,8 +122,12 @@ private function updateRepr(): void
         if ($punctuation = '.' === $last || '?' === $last ? $last : '') {
             $this->message = substr($this->message, 0, -1);
         }
-        if ($this->source && $this->source->getName()) {
-            $this->message .= \sprintf(' in "%s"', $this->source->getName());
+        if ($this->source) {
```

Comment:
This change looks wrong to me. Here, we want to use the name of the template, not the path. The path/line information is part of the regular Exception properties.

Reply:
But we need to do that to make Symfony works. I don't know how we can do better.